src/CattyTests/Resources/ParserTests/Custom/ValidProjectAllBricks/ValidProjectAllBricks0993.xml

```diff
@@ -1,765 +1,755 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
 <program>
-  <header>
```

Comment:
Please do not change formatting of this file, this makes the review really difficult. Please revert back to 2 spaces instead of 4.